forge/test/mlir/test_ops.py

```diff
@@ -13,6 +13,48 @@
 from forge.op.eval.common import compare_with_golden_pcc, compare_with_golden
 from forge.tensor import to_forge_tensors, to_pt_tensors
 
+
+@pytest.mark.parametrize("operand_and_cast_dtype", [(torch.float32, torch.int32), (torch.int32, torch.float32)])
```

Comment:
@chandrasekaranpradeep can you create issues to test out cast op with other dtypes and assignee it to the Bringup milestone? Just as reminder to check other cases as well (no need to push it with this change)

Reply:
Sure Nikola.
I will create a issue for cast op for checking other dtypes with Bringup milestone.
FYI, There is issue for the cast support in forge (i.e https://github.com/tenstorrent/tt-forge-fe/issues/336) I will link the issue in this PR for current implementation I have done for cast op

Reply:
Created issue for cast op: https://github.com/tenstorrent/tt-forge-fe/issues/610